src/App.js

```diff
@@ -41,11 +43,29 @@ function App() {
   }, [])
   console.log("dogs", dogs)
 
+  function getTicket() {
+    const url = `${process.env.REACT_APP_FETCH_URL}/tickets`;
+    fetch(url)
+      .then((res) => res.json())
+      .then((data) => {
+        console.log("Incoming Tickets Data: ", data)
```

Comment:
Might want to remove this console log now that you've finished this work in order to keep the console clear :)